Generators/include/Generators/PrimaryGenerator.h

```diff
@@ -55,6 +55,19 @@ class PrimaryGenerator : public FairPrimaryGenerator
       **/
   Bool_t GenerateEvent(FairGenericStack* pStack) override;
 
+  /** Public method AddTrack
+      Adding a track to the MC stack. To be called within the ReadEvent
+      methods of the registered generators.
+      *@param pdgid Particle ID (PDG code)
+      *@param px,py,pz Momentum coordinates [GeV]
+      *@param vx,vy,vz Track origin relative to event vertex
+      **/
+  virtual void AddTrack(Int_t pdgid, Double_t px, Double_t py, Double_t pz,
```

Comment:
no virtual necessary (since we use override)

Reply:
indeed, copy-paste mistake.